content/en/hosting-and-deployment/hosting-on-github.md

```diff
@@ -55,7 +55,7 @@ name: github pages
 on:
   push:
     branches:
-      - main  # Set a branch to deploy
+      - master  # Set a branch to deploy
```

Comment:
HugoDocs uses 'master', but the default new branch name on GitHub is 'main'.
Main is quickly becoming the standard so I think we should keep it as that.